tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/BlockListEditor/blockListEditorContent.spec.ts

```diff
@@ -0,0 +1,684 @@
+    await expect(page.locator('.umb-block-list__block--view').nth(0)).toHaveText(elementName);
+
+    // Clean
+    await umbracoApi.documentTypes.ensureNameNotExists(documentName);
+    await umbracoApi.documentTypes.ensureNameNotExists(elementName);
+    await umbracoApi.dataTypes.ensureNameNotExists(blockListName);
+  });
+
+  test('can update content with a block list editor', async ({page, umbracoApi, umbracoUi}) => {
+    await umbracoApi.documentTypes.ensureNameNotExists(documentName);
+    await umbracoApi.documentTypes.ensureNameNotExists(elementName);
+    await umbracoApi.dataTypes.ensureNameNotExists(blockListName);
+    await umbracoApi.content.deleteAllContent();
+
+    await createContentWithOneBlockListEditor(umbracoApi, documentName, documentAlias, elementName, elementAlias, blockListName, blockListAlias);
+
+    await umbracoUi.goToSection(ConstantHelper.sections.content);
+    await umbracoUi.refreshContentTree();
+
+    // Opens the content with the block list editor
+    await page.locator('[data-element="tree-item-' + blockListName + '"]').click();
+
+    // Updates the block list editor inside of the content
+    await page.locator('[ui-sortable="vm.sortableOptions"]').nth(0).click();
+    // Updates content
+    await page.locator('[id="sub-view-0"]').locator('[id="title"]').fill('ContentTest');
+    await page.locator('[data-element="sub-view-settings"]').click();
+    // Adds text to the setting element
+    await page.locator('[id="sub-view-1"]').locator('[id="title"]').fill('SettingTest');
+    await page.locator('[label="Submit"]').click();
+    await umbracoUi.clickElement(umbracoUi.getButtonByLabelKey(ConstantHelper.buttons.saveAndPublish));
+
+    // Assert
+    await umbracoUi.isSuccessNotificationVisible();
+
+    // Clean
+    await umbracoApi.documentTypes.ensureNameNotExists(documentName);
+    await umbracoApi.documentTypes.ensureNameNotExists(elementName);
+    await umbracoApi.dataTypes.ensureNameNotExists(blockListName);
+    await umbracoApi.content.deleteAllContent();
+  });
+
+  test('can delete a block list editor in content', async ({page, umbracoApi, umbracoUi}) => {
+    await umbracoApi.documentTypes.ensureNameNotExists(documentName);
+    await umbracoApi.documentTypes.ensureNameNotExists(elementName);
+    await umbracoApi.dataTypes.ensureNameNotExists(blockListName);
+    await umbracoApi.content.deleteAllContent();
+
+    await createContentWithOneBlockListEditor(umbracoApi, documentName, documentAlias, elementName, elementAlias, blockListName, blockListAlias);
+
+    await umbracoUi.goToSection(ConstantHelper.sections.content);
+    await umbracoUi.refreshContentTree();
+
+    // Opens the content with the block list editor
+    await page.locator('[data-element="tree-item-' + blockListName + '"]').click();
+
+    // Deletes the block list editor inside of the content
+    await page.locator('[title="Delete"]').click();
+    await page.locator('[label-key="actions_delete"]').click();
```

Comment:
we should be using the constant from uiHelper to find button by label key `GetByLabelKey` and use the ConstantHelper action - `actions-delete`